python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -1843,15 +1843,14 @@ def check_to_local_iterator_not_fully_consumed(self):
         self.assertEqual(df.take(8), result)
 
     def test_same_semantics_error(self):
-        with QuietTest(self.sc):
```

Comment:
this checking only happens in python side, no need to use `QuietTest` which configs log4j